### PR TITLE
Extend the section about the team's responsibilities

### PR DESCRIPTION
**Pull request by peti**

### CODE REVIEW

File: rfcs/0020-security-on-call.md [hunk 1]
```diff
@@ -58,12 +60,31 @@ experienced community members for help.
...
+I propose we create a 24 hour on-call rotation of volunteers who will be
+responsible for handling any publicly disclosed security issues that
+arise during their shift, where "handling an issue" means:
+
+ - Create a ticket for the given issue to ensure that others can
+   see, follow, and contribute to the current state of affairs in
+   a coordinated manner.
+
+ - Determine whether Nixpkgs and/or NixOS is vulnerable to the given
+   vulneravbility and record your findings in the appropriate ticket.
```
Comment (grahamc) [inline rfcs/0020-security-on-call.md]: nit: vulnerability

**Comment by grahamc** [comment]
Looks like excellent additions, a minor nit and I'll merge.

**Comment by peti** [comment]
I fixed the typo.

**Comment by grahamc** [comment]
Thank you!

**Pull request merged**